.circleci/config.yml

```diff
@@ -0,0 +1,105 @@
+version: 2.1
+executors:
+  default-executor:
+    docker:
+      - image: "cimg/node:12.22.11-browsers"
+    working_directory: ~/project
+    resource_class: medium
+
+commands:
+  restore_test_cache:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-deps-{{ checksum "package-lock.json" }}
+            - v1-deps-
+  save_test_cache:
+    steps:
+      - save_cache:
+          key: v1-deps-{{ checksum "package-lock.json" }}
+          paths:
+            - node_modules
+            - ~/.cache/ms-playwright
+
+  setup:
+    steps:
+      - run:
+          name: Install dependencies
+          command: |
+            npm --production=false install
+            npm --production=false update
```

Comment:
Oof... this `update` is a little embarrassing in retrospect. I'll clean it up when I do the rest of the modernizing, though :)